test/test_autograd.py

```diff
@@ -5635,11 +5635,6 @@ def test_leaky_relu_inplace_with_zero_or_neg_slope(self, device):
         with self.assertRaisesRegex(RuntimeError, "call out-of-place version"):
             b.backward(torch.ones(2, device=device))
 
-        a = torch.tensor([-2., 0., 2.], device=device, requires_grad=True)
```

Comment:
I think this is worth keeping!
We do rely on this subgradient being 0. Even though we most likely will never willingly change that. It is good to have a test that makes sure it does not happen by mistake.

Reply:
sure. I will make a separate test to test this.